src/main/kotlin/no/nav/familie/ef/personhendelse/inntekt/InntektsendringerService.kt

```diff
@@ -167,6 +213,33 @@ class InntektsendringerService(
         oppgaveClient.leggOppgaveIMappe(oppgaveId, "63") // Inntektskontroll
     }
 
+    private fun opprettOppgaveForUføretrygEndring(
+        inntektOgVedtakEndring: InntektOgVedtakEndring,
+        beskrivelse: String,
+    ) {
+        val oppgaveId =
+            oppgaveClient.opprettOppgave(
+                OpprettOppgaveRequest(
+                    ident =
+                        OppgaveIdentV2(
+                            ident = inntektOgVedtakEndring.personIdent,
+                            gruppe = IdentGruppe.FOLKEREGISTERIDENT,
+                        ),
+                    saksId = null,
+                    tema = Tema.ENF,
+                    oppgavetype = Oppgavetype.VurderKonsekvensForYtelse,
+                    fristFerdigstillelse = fristFerdigstillelse(), // TODO: Sjekk hva som ønskes her
+                    beskrivelse = beskrivelse,
+                    enhetsnummer = "4489",
+                    behandlingstema = null, // Gjelder-feltet i Gosys
```

Comment:
Ser det er satt til null i vanlig inntektsendringsoppgaver også, men gjerne undersøk om denne skal settes til for eksempel Overgangsstønad